src/scss/_degrees.scss

```diff
@@ -157,3 +157,7 @@
   width: 1em;
   margin-left: -1em;
 }
+
+.program-length-image {
```

Comment:
Is this class supposed to be .icon-calendar?  .program-length-image doesn't appear to be in use anywhere.